### ISSUE TITLE
Cannot specify epoch number in librispeech tutorial

### ISSUE THREAD

**Issue by nchp89** [facebookresearch/wav2letter#753]
I am working through the librispeech tutorial but am unable to train the model for 25 epochs as instructed. 

The tutorial states: **Train the model for 25 epochs (specified by -iter flag) **

However, the --iter flag seems to only specify the number of updates. I've looked through the training documentation but cannot find any flag to specify epoch number. 
*Note: I set --iter to 1000 this time

train.cfg
```
--datadir=/my_data/data
--rundir=/my_data/data
--archdir=/my_data
--train=lists/train-clean-100.lst
--valid=lists/dev-clean.lst
--input=flac
--arch=network.arch
--tokens=/my_data/data/am/tokens.txt
--lexicon=/my_data/data/am/lexicon.txt
--criterion=ctc
--lr=0.1
--maxgradnorm=1.0
--replabel=1
--surround=|
--onorm=target
--sqnorm=true
--mfsc=true
--filterbanks=40
--nthread=4
--batchsize=4
--runname=librispeech_clean_trainlogs
--iter=1000
--logtostderr=1
```

log
```
epoch:        1 | nupdates:         1001 | lr: 0.100000 | lrcriterion: 0.000000 | runtime: 00:30:30 | bch(ms): 1828.97 | smp(ms): 0.40 | fwd(ms): 611.09 | crit-fwd(ms): 34.70 | bwd(ms): 1136.82 | optim(ms): 74.23 | loss:   40.29816 | train-TER: 99.78 | train-WER: 99.99 | lists/dev-clean.lst-loss:   27.94572 | lists/dev-clean.lst-TER: 96.23 | lists/dev-clean.lst-WER: 98.25 | avg-isz: 1243 | avg-tsz: 208 | max-tsz: 317 | hrs:   13.83 | thrpt(sec/sec): 27.20
```

Am I missing something here? How can I specify the number of epochs for training?

**Comment by tlikhomanenko** [comment]
Hi @nchp89,

we have recently change `--iter` to be number of updates instead of epochs and in on the way fixing the tutorial regarding these changes. We don't have parameter to specify number of epochs now, because a lot of things are computed based on updates, like learning rate schedules, that is why we switched to updates. You can simply compute how many updates you have in the epoch and just multiply this number by 25. Or just set some large number as soon as you have fixed lr all the time.
